extensions/wikia/UserRenameTool/UserRenameToolController.class.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+class UserRenameToolController extends SpecialPage {
+	private $oldUsername;
+	private $newUsername;
+	private $reason;
+	private $token;
+	private $notifyRenamed;
+	private $confirmAction;
+	private $showConfirm = true;
+
+	private $errors = [];
+	private $warnings = [];
+	private $info = [];
+
+	public function __construct() {
+		parent::__construct( 'UserRenameTool', 'renameuser', true );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "renameuser" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)